### PR TITLE
Feature: Datasource for looker_group

### PR DESCRIPTION
**Pull request by jessicasomaiya**
## Description 

This is a datasource to retrieve an existing `looker_group`.

### CODE REVIEW

File: looker/data_source_looker_group_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+				data "looker_group" "all" {
+					name = "All Users"
+				}
+
+				output "looker_group_all_id" {
+					value = data.looker_group.all.id
+				}
+				`,
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckOutput("looker_group_all_id", "1"),
```
Comment (Vitzeno) [inline looker/data_source_looker_group_test.go:25]: Is the ID for `all_users` always going to be 1 or should we manually check that?

Comment (Vitzeno) [reply looker/data_source_looker_group_test.go:25]: If it's always 1 then I think this is fine

Comment (mattoddie) [reply looker/data_source_looker_group_test.go:25]: @Vitzeno always seems to be 1. In the UI it shows as a system group that is locked so think this is fine

**Review by mattoddie — APPROVED** [review]

File: looker/data_source_looker_group_test.go (hunk 1 above)
Comment (jessicasomaiya) [reply looker/data_source_looker_group_test.go:25]: Ahh good point, I think it is because that group is generated by looker 🤞🏾

**Pull request merged**